tutorials/01-pymapdl.py

```diff
@@ -0,0 +1,255 @@
+"""
+.. _ref_pymapdl_tutorial:
+
+================
+PyMAPDL Tutorial
+================
+
+
+In this tutorial, we will introduce some of the capabilities of
+PyMAPDL applied to Aerospace students and advocates.
+
+"""
+
+#########################################################
+# Python + Ansys = PyAnsys
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Pyansys is Ansys effort on making their products accessible
+# using Python programming language.
+#
+# Today we will focus on PyMAPDL with is the Python library
+# which exposes MAPDL (Ansys Structural)
+#
+
+##########################################################
+# PyMAPDL
+# ~~~~~~~
+# MAPDL provides many tools and features which would require
+# A LOT of time to explain, so let's have a quick overview
+
+##########################################################
+# Launching PyMAPDL
+# -----------------
+# But first, let's launch PyMAPDL.
+from ansys.mapdl.core import launch_mapdl
+
+mapdl = launch_mapdl()
+
+print(mapdl)
+
+############################################################
+# Geometry
+# --------
+#
+# PyMAPDL support points (keypoints), lines, areas, and volumes
+# for geometry definition.
```

Comment:
Strongly considering to do a small demo with PyGeometry... :smile:

Reply:
LOOOOOOOOOOOOOOLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLL

You don't wanna go thereee... xD